fec_eregs/static/regulations/css/less/module/custom.less

```diff
@@ -105,25 +105,49 @@ a {
 }
 
 .main-head {
-  border-bottom: 3px solid @primary;
+  border-bottom: none;
+  height: @mainhead_height;
 }
 
 .sub-head {
   background-color: @subhead-background;
-  border-bottom: @subhead-secondary-border;
-  color: @primary;
+  border-bottom: none;
+  color: @gray-lightest;
+  height: @subhead_height;
+  line-height: inherit;
 }
 
 
 // TOC header
+// TODO We're really pushing the limits here, our TOC is diverged
+// significantly from regulations-site and should probably be written
+// with completely fec-eregs specific styles.
 .toc-head {
   background-color: @toc-background;
-  border-bottom: @subhead-secondary-border;
+  border-bottom: none;
+  border-right: none;
+  height: auto;
+  left: 188px;
+  position: fixed;
+  top: @mainhead_height + @subhead_height;
+  transition: none;
+  width: auto;
+  z-index: 1;
 
   a.current,
   a:hover {
     background-color: @primary;
   }
+
+  #panel-link {
```

Comment:
Hmm... The nature of these styles are that they are overriding `regulations-site`, so they should have strong-ish specificity to avoid getting broken in a future update of `regulations-site` but it's still not a guarantee. It might just be better to use classes as you point out.